common/static/sass/_mixins-inherited.scss

```diff
@@ -118,7 +118,7 @@
 // ====================
 
 // mixins - grandfathered
-@mixin button {
+@mixin button-sunsetted {
```

Comment:
NOTE: this homegrown Studio ``button()`` mixin from very early on had a conflicting name. Since all of the mixins contained in this file are pretty much ones we do not rely on when building new UI and are due to be sunsetted, I've added a suffix to avoid the Bourbon deprecated ``button()`` mixin being called mistakenly when this is included in further styles.